### DISCUSSION TITLE
Dropdown 样式如何和antd差不多

### DISCUSSION THREAD

**Discussion by Julycc41** [ant-design/ant-design-mobile#5634] [Q&A]
现在antd-mobile 下拉框是一个modal 弹出 如何整成antd的，因为我是需要做多个下拉框 多个看起来很怪
类似于antd 中
![1662090474915](https://user-images.githubusercontent.com/80231740/188055244-a5d7e802-f91b-4da9-ae6b-22ed3db0b5d8.png)
![1662090491586](https://user-images.githubusercontent.com/80231740/188055271-77c2dae7-f063-47d6-abd8-d9e197ca7069.png)

**Comment by awmleer** [comment]
这两个是完全不一样的两个组件，所以即便做样式调整，也很难调整成一样的

你可以考虑直接使用 antd 的组件？
